linux/nvme.h

```diff
@@ -525,7 +525,8 @@ struct nvme_smart_log {
 	__u8			avail_spare;
 	__u8			spare_thresh;
 	__u8			percent_used;
-	__u8			rsvd6[26];
+	__u8			endu_grp_crit_warn_sumry;
+	__u8			rsvd6[25];
```

Comment:
this should be renamed to rsvd7 since it's occupying the 7th byte.

Reply:
Has been fixed and pushed